components/typeahead/typeahead.directive.ts

```diff
@@ -6,13 +6,13 @@ import {
 } from 'angular2/core';
 import {NgModel}from 'angular2/common';
 
+import {Observable, Subject} from 'rxjs/Rx';
```

Comment:
I don't think we should import the whole rxjs library here.
It's better to import them separatly, that way we don't include more code than we need.

```
import {Observable} from 'rxjs/Observable';
import {Subject} from 'rxjs/Subject';
```